cmd/nknc/commands/asset.go

```diff
@@ -0,0 +1,216 @@
+package commands
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/howeyc/gopass"
+	api "github.com/nknorg/nkn/v2/api/common"
+	"github.com/nknorg/nkn/v2/api/httpjson/client"
+	"github.com/nknorg/nkn/v2/common"
+	"github.com/nknorg/nkn/v2/config"
+	"github.com/nknorg/nkn/v2/util/log"
+	"github.com/nknorg/nkn/v2/vault"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	RANDBYTELEN = 4
+)
+
+// assetCmd represents the asset command
+var assetCmd = &cobra.Command{
+	Use:   "asset",
+	Short: "asset registration, issuance and transfer",
+	Long:  "",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		showusage := true
+		cmd.Flags().Visit(func(name *pflag.Flag) {
+			showusage = false
+		})
+		if showusage {
+			return cmd.Usage()
+		}
+		if err := assetAction(); err != nil {
+			log.Error(err)
+		}
+		return nil
+	},
+}
+
+var (
+	issue     bool
+	transfer  bool
+	to        string
+	value     string
+	fee       string
+	assetname string
+	symbol    string
+	precision uint32
+)
+
+func init() {
+	rootCmd.AddCommand(assetCmd)
+
+	assetCmd.Flags().BoolVarP(&issue, "issue", "i", false, "issue asset")
+	assetCmd.Flags().BoolVarP(&transfer, "transfer", "t", false, "transfer asset")
+	assetCmd.Flags().StringVarP(&WalletFile, "wallet", "w", "", "wallet name")
+	assetCmd.Flags().StringVar(&WalletPassword, "password", "", "wallet password")
```

Comment:
There is default value for `WalletFile` from `config.Parameters.WalletFile`.
`assetCmd` used global var will lost  the default value.

Not only `WalletFile` flag in `assetCmd`, but also several flags in all `sub commands` lost their default value.